frontend/components/HtmlReferenceContainer/components/YotpoWidget/index.jsx

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import styles from './style';
+import getYotpo from '../../../../helpers/getYotpo';
+
+/**
+ * Mounts yotpo script.
+ */
+class YotpoScript extends Component {
+  static propTypes = {
+    config: PropTypes.shape().isRequired,
+    productId: PropTypes.string.isRequired,
+    product: PropTypes.shape(),
+  };
+
+  static defaultProps = {
+    product: null,
+  }
+
+  /**
+   * Constructs
+   * @param {Objects} props props
+   */
+  constructor(props) {
+    super(props);
+    this.yotpoMountTimeout = undefined;
+    this.yotpoRefreshTimeout = undefined;
+  }
+
+  /**
+   * Mounts Yotpo script.
+   */
+  componentDidMount() {
+    if (!window.yotpo) {
+      getYotpo(this.props.config.yotpoAppKey);
+      return;
+    }
+    window.yotpo.initialized = false;
+    window.yotpo.clean();
+    this.yotpoMountTimeout = setTimeout(() => {
+      window.yotpo.initWidgets();
+      this.yotpoMountTimeout = undefined;
+    }, 500);
+  }
+
+  /**
+   * Cleanup timeout request
+   */
+  componentWillUnmount() {
+    if (this.yotpoMountTimeout) {
+      clearTimeout(this.yotpoMountTimeout);
+    }
+    if (this.yotpoRefreshTimeout) {
```

Comment:
After some of your changes this is always undefined now. Did you forget to remove this?

Reply:
I must've forgot to remove this or lost the commit. Updated.